lib/msal-browser/src/interaction_handler/IInteractionHandler.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { AuthorizationCodeModule, AuthenticationParameters } from "msal-common";
+import { BrowserStorage } from "../cache/BrowserStorage";
+
+export abstract class IInteractionHandler {
```

Comment:
does this abstraction hold true for interactions in general? is there carry over to popup? How will it translate to node scenarios?

Reply:
This abstraction is specific to browsers (which is why it is defined in the browser class). After our discussions today with the Azure SDK team, this may change, but for now this is only defining interactions in a browser. Yes, there will be carryover to popup in the next PR.